.github/workflows/veramo-website.yml

```diff
@@ -0,0 +1,20 @@
+name: Trigger veramo-website build
+on:
+  push:
+    branches:
+      - "main"
+      - "italo/fix-docs"
+permissions: write-all
```

Comment:
I think permissions alterations should not be necessary as long as the GH_TOKEN has the required scope.

Reply:
Removed.